network-api/networkapi/buyersguide/templates/category_page.html

```diff
@@ -19,7 +19,7 @@ <h1 class="h2-heading">{{ category.name }}</h1>
         <div class="row">
           {% for product in products %}
           <div class="category-item-container col-6 {% if product.draft %} draft-product{% endif %}{% if product.adult_content %} adult-content{% endif %}">
-            <a href="/en/privacynotincluded/products/{{ product.slug }}">
+            <a href="/privacynotincluded/products/{{ product.slug }}">
```

Comment:
if I understand the documentation correctly, we might be able to use `<a href="{{ product.url }}">` here, and wagtail will automatically use the correct relative url because it's internal... we should test that, because if it is a whole bunch of our urls can be turned into single `blah.url` instructions rather than partially hardcoding them.

Reply:
(If I understand the following line correctly) `Product` is not a Page object so I'm afraid we can't use pageurl on it.

```
class Product(models.Model):
```

Reply:
Got errors when I tried it

```py
TemplateSyntaxError at /en/privacynotincluded/categories/toys-games/

Could not parse the remainder: ' product' from 'page_url product'
```

Reply:
```py
{% url 'product-view' product.slug %}
```
seems to work so I'll use this instead.

Reply:
ah, good finds!